lib/turbo_reflex/base.rb

```diff
@@ -1,29 +1,36 @@
 # frozen_string_literal: true
 
-# Reflex instances have access to the following methods and properties.
+# TurboReflex base superclass.
+# All TurboReflex classes should inherit from this class.
 #
-# * controller ........ The Rails controller processing the HTTP request
-# * element ........... A struct that represents the DOM element that triggered the reflex
-# * hijack_response ... Hijacks the response, must be called last (halts further request handling by the controller)
-# * params ............ Reflex specific params (frame_id, element, etc.)
-# * render ............ Renders Rails templates, partials, etc. (doesn't halt controller request handling)
-# * renderer .......... An ActionController::Renderer
-# * turbo_stream ...... A Turbo Stream TagBuilder
-# * turbo_streams ..... A list of Turbo Streams to append to the response
+# Reflexes are executed via a before_action in the Rails controller lifecycle.
+# They have access to the following methods and properties.
+#
+# * dom_id ....................... The Rails dom_id helper
+# * dom_id_selector .............. Returns a CSS selector for a dom_id
+# * controller ................... The Rails controller processing the HTTP request
+# * element ...................... A struct that represents the DOM element that triggered the reflex
+# * params ....................... Reflex specific params (frame_id, element, etc.)
+# * render ....................... Renders Rails templates, partials, etc. (doesn't halt controller request handling)
+# * render_response .............. Renders a full controller response
+# * renderer ..................... An ActionController::Renderer
+# * prevent_controller_action .... Prevents the rails controller/action from running (i.e. the reflex handles the response entirely)
+# * turbo_stream ................. A Turbo Stream TagBuilder
+# * turbo_streams ................ A list of Turbo Streams to append to the response (also aliased as streams)
 #
 class TurboReflex::Base
   class << self
-    def response_hijackers
-      @response_hijackers ||= Set.new
+    def preventers
+      @preventers ||= Set.new
     end
 
-    def hijack_response(options = {})
-      response_hijackers << options.with_indifferent_access
+    def prevent_controller_action(options = {})
+      preventers << options.with_indifferent_access
     end
```

Comment:
👍🏻 I like this method name change. It's much more clear as to what's actually happening.